src/nvidia-cuda/install.sh

```diff
@@ -69,7 +71,15 @@ apt-get install -yq "$cuda_pkg"
 
 if [ "$INSTALL_CUDNN" = "true" ]; then
     # Ensure that the requested version of cuDNN is available AND compatible
-    cudnn_pkg_version="libcudnn8=${CUDNN_VERSION}-1+cuda${CUDA_VERSION}"
+    #if major cudnn version is 9, then we need to install libcudnn9-cuda-<major_version> package
+    #else we need to install libcudnn8-cuda-<major_version> package
+    if [[ $major_cudnn_version = "9" ]]
```

Comment:
Can we check for 9 and greater? So that we don't need more changes when v10 is released

Reply:
Done